EE_Event_Cart_Line_Item_Display_Strategy.php

```diff
@@ -271,45 +271,55 @@ private function _ticket_row( EE_Line_Item $line_item, $options = array() ) {
      * @return mixed
      */
 	private function _ticket_qty_input( EE_Line_Item $line_item, EE_Ticket $ticket, $required = '' ) {
-		if ( $ticket->remaining() - $line_item->quantity() ) {
-			$disabled = '';
-			$disabled_class = '';
-			$disabled_title = __( 'add one item', 'event_espresso' );
-			$query_args = array( 'event_cart' => 'add_ticket', 'ticket' => $ticket->ID(), 'line_item' => $line_item->code() );
+        $input_disabled_class = '';
+        $input_disabled = '';
+
+		if ( !($ticket->remaining() - $line_item->quantity()) || ($line_item->quantity() >= $ticket->max()) ) {
```

Comment:
this is called more than a few times in this method so would be good to extract to a variable like:

```
```suggestion
		$line_item_quantity = $line_item->quantity();
		if ( !($ticket->remaining() - $line_item_quantity) || ($line_item_quantity >= $ticket->max()) ) {
```

same goes with `$line_item->code()` although I know that you didn't touch that in this PR, but making that change as well would be great